src/pages/stacks-blockchain/accounts.md

```diff
@@ -0,0 +1,175 @@
+
+Assets cannot leave an account without an action from the account owner. All changes to assets (and the balances of the account) require a corresponding transaction.
+
+-> The [transaction type](/stacks-blockchain/transactions#types) doesn't need to be a token transfer - contract deploy and contract call transactions can change the balances of an account
+
+## Creation
+
+When an account is created, a 24-phrase mnemonic is generated. This is often referred to as the **seed phrase**. The seed phrase provides access to Stacks 2.0 accounts.
+
+!> If the seed phrase is lost, access to the associated account cannot be restored. No person or organization, including Blockstack, can recover a lost seed phrase.
+
+The easiest way to generate a new Stacks 2.0 account is to use the [Stacks 2.0 CLI](https://github.com/blockstack/cli-blockstack/tree/feature/stacks-2.0-tx):
+
+```shell
+# install CLI globally
+sudo npm install -g "https://github.com/blockstack/cli-blockstack#feature/stacks-2.0-tx"
+
+# generate a new account and store details in a new file
+# '-t' option makes this a testnet account
+blockstack make_keychain -t > cli_keychain.json
+```
+
+The `make_keychain` command will create following file:
+
+```js
+{
+  "mnemonic": "aaa bbb ccc ddd ...",
+  "keyInfo": {
+    "address": "STJRM2AMVF90ER6G3RW1QTF85E3HZH37006D5ER1",
+    "privateKey": "5a3f1f15245bb3fb...",
+    "index": 0,
+    "btcAddress": "biwSd6KTEvJcyX2R8oyfgj5REuLzczMYC1"
+  }
+}
+```
+
+| Field                | Description                                                                                                                                                          |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `mnemonic`           | A 24-phrase seed phrase used to access the account, generated using [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) with 256 bits of entropy |
+| `keyInfo.address`    | Stacks address for the account                                                                                                                                       |
+| `keyInfo.privateKey` | Private key for the account. Required for [token transfers](/stacks-blockchain/transactions#stacks-token-transfer) and often referred to as `senderKey`              |
+| `keyInfo.index`      | Nonce for the account, starting at 0                                                                                                                                 |
+| `keyInfo.btcAddress` | Corresponding BTC address for the account. A construct from the previous blockchain (Stacks 1.0), not used at the moment.                                            |
+
+Note that the new account is instantiated implicitly and does not require internet access. There is no need to manually instantiate it on the Stacks 2.0 blockchain.
```

Comment:
I believe the question asked was: "do account need to be created on the blockchain explicitly?". without the context, it might be odd though. how would you reword it?

Reply:
Maybe: A new address/account automatically exists for each new private key. There is no need to manually instantiate an account on the Stacks 2.0 blockchain.

Reply:
I like it! thanks!